apps/remote_control/lib/lexical/remote_control/api.ex

```diff
@@ -16,6 +16,21 @@ defmodule Lexical.RemoteControl.Api do
   defdelegate schedule_compile(project, force?), to: Build
   defdelegate compile_document(project, document), to: Build
 
+  def maybe_schedule_compile(project, triggered_file_uri, triggered_message) do
+    RemoteControl.call(project, Build, :maybe_schedule_compile, [
+      triggered_file_uri,
+      triggered_message
+    ])
+  end
+
+  def maybe_compile_document(project, document, triggered_message) do
+    RemoteControl.call(project, Build, :maybe_compile_document, [
+      project,
+      document,
+      triggered_message
+    ])
+  end
```

Comment:
Really? yes, it can, but we have two `if else` statements there. If we delegate, that means we would need to write more `RemoteControl.call` statements, which would make the code less clear.